app/src/main/res/layout/recycler_item.xml

```diff
@@ -1,32 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Go through the whole file to see if there's color, dimensions or other info you can extract to clean up